jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/server/Dispatcher.java

```diff
@@ -144,19 +146,16 @@ private static ActionProcessor chooseProcessor(HttpAction action) {
 
         // ---- Determine Endpoint.
         String endpointName = mapRequestToEndpointName(action, dataAccessPoint);
+        // Main step of choosing the endpoint for the dispatch of the request.
+        // An endpoint is a (name, operation).
+        // There may be multiple operations for an endpointName of this data service.
 
         Endpoint endpoint = chooseEndpoint(action, dataService, endpointName);
-        if ( endpoint == null ) {
-            if ( isEmpty(endpointName) )
-                ServletOps.errorBadRequest("No operation for request: "+action.getActionURI());
-            else {
-                // No dispatch - the filter passes these through if the ActionProcessor is null.
-                return null;
-                // If this is used, resources (servlets, sttaic files) under "/dataset/" are not accessible.
-                //ServletOps.errorNotFound("No endpoint: "+action.getActionURI());
-            }
+        if ( endpoint == null )
+            // Named service, no such endpoint.
+            // Allows for resources under /dataset/
+            // Does to Jetty's default handling (404 for GET, 405 other methods).
```

Comment:
`Does` -> `Goes`

Reply:
This is actually out of date. This PR also changes the default handling of this case, replacing the Jetty default handling because Jetty is different to other servers.